spec/cucumber/core/event_bus_spec.rb

```diff
@@ -0,0 +1,151 @@
+require "cucumber/core/event_bus"
+
+module Cucumber
+  module Core
+    module Events
+
+      class TestEvent < Core::Event.new(:some_attribute)
+      end
+
+      AnotherTestEvent = Core::Event.new
+
+      UnregisteredEvent = Core::Event.new
+    end
+
+    describe EventBus do
+      let(:bus) { EventBus.new(registry) }
```

Comment:
Thanks - great feedback. I agree, let's make it unambiguous. I'll change it to `event_bus` everywhere.